mkfs/rootdir.c

```diff
@@ -701,6 +705,47 @@ static int ftw_add_inode(const char *full_path, const struct stat *st,
 	return 0;
 };
 
+static int set_default_subvolume(struct btrfs_trans_handle *trans)
+{
+	struct btrfs_path path = { 0 };
+	struct btrfs_dir_item *di;
+	struct btrfs_key location;
+	struct extent_buffer *leaf;
+	struct btrfs_disk_key disk_key;
+	u64 features;
+
+	di = btrfs_lookup_dir_item(trans, trans->fs_info->tree_root, &path,
+				   btrfs_super_root_dir(trans->fs_info->super_copy),
+				   "default", 7, 1);
+	if (!di || IS_ERR(di)) {
```

Comment:
You can use IS_ERR_OR_NULL here

Reply:
Thanks, I've pushed a version with this change.